tests/testthat/test-vdiagram.R

```diff
@@ -19,4 +19,20 @@ test_that("vdiagram examples work", {
  expect_is(vdiagram(data, 'html'), 'character')
 })
 
+context("testing vdiagram when gage heights are blank")
+test_that("vdiagram examples work", {
```

Comment:
I think we are trying to add less JSON files to pinpoint what part of the JSON is being tested. See `test-utils-json.R`. I don't think it's necessary to change this one, but just so you know in the future!